javaparser-core/src/main/javacc/java.jj

```diff
@@ -714,56 +714,69 @@ StubUnit StubUnit():
 }
 {
     ( LOOKAHEAD(2) ";" )*
-    [ LOOKAHEAD(PackageDeclaration()) pakage = PackageDeclaration() ]
+    /*
+     * Stub file format described in the Checker Framework Manual:
+     * https://checkerframework.org/manual/#stub-format .
+     * All imports must be at the beginning of the file.
```

Comment:
Delete the next three lines; it's adequate to refer to the manual.
Should we relax the restriction that all imports must be at the beginning of the file?
I think you have already done this, in which case you can delete the restriction  from the Checker Framework manual.